packages/core/src/utils/common.js

```diff
@@ -65,3 +65,7 @@ export function getMemberExressionTarget(node: MemberExpression): Node {
   } while (target.type === NODE.MEMBER_EXPRESSION);
   return target;
 }
+
+export function ensureArray<T>(value: T | Array<T>): Array<T> {
+  return Array.isArray(value) ? value : [value];
```

Comment:
Can we return an empty Array, if the `value` is `undefined` or `null`? It's micro-optimization, but we will process twice fewer elements for each node 😃

Reply:
yeh good point
this will also allow removing the check in `isFunction` that you commented before
will do

Reply:
done